### PR TITLE
Update client README and commit generated models

### PR DESCRIPTION
**Pull request by gabrielmoncea**

### CODE REVIEW

File: packages/amplication-client/README.md [hunk 1]
```diff
@@ -15,27 +15,65 @@ If you need help or have a question, go to our [Discord channel](https://discord
...
 ## Available Scripts
 
-In the project directory, you can run:
+In the project directory, after you setup the server, you can run:
 
 ### `npm start`
 
 Runs the app in the development mode.<br />
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+Open [http://localhost:3001](http://localhost:3001) to view it in the browser.
```
Comment (yuval-hazaz) [inline packages/amplication-client/README.md:76]: Good catch!

**Review by yuval-hazaz — CHANGES_REQUESTED** [review]
Please do not include the changes in the package-lock and models.ts in this PR
Please direct the user to the existing instructions document to keep single source

**Comment by gabrielmoncea** [comment]
> Please do not include the changes in the package-lock and models.ts in this PR
> Please direct the user to the existing instructions document to keep single source

Done!

**Review by yuval-hazaz — APPROVED** [review]

**Comment by yuval-hazaz** [comment]
> > Please do not include the changes in the package-lock and models.ts in this PR
> > Please direct the user to the existing instructions document to keep single source
> 
> Done!

Congrats on your first merge!

**Comment by yuval-hazaz** [comment]
@all-contributors please add @gabrielmoncea  for code

**Pull request merged**